README.md

```diff
@@ -177,6 +177,15 @@ the "samba" container that refers to it by name. So, in this example, if you
 were on OS-X you now have /Volumes/data and /data in container being shared. You
 can change the paths as needed.
 
+##### Installing secure Registry certificates
+
+From Boot2Docker version 1.6.0, you can add your Registry server's public certificate
```

Comment:
I'd prefer if this just documented master instead, since we don't officially support any version other than latest (we'll never release another update to the 1.3.3 ISO, for example).

ie, start directly with "You can add your Registry server's public ..."